tsconfig.json

```diff
@@ -8,8 +8,6 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "skipLibCheck": true,
-    "declaration": true,
-    "declarationDir": "bin",
```

Comment:
So we can delete these because pika takes care of generating types via `@pika/plugin-build-types`?

Reply:
Yup!